phpBB/phpbb/event/php_exporter.php

```diff
@@ -264,7 +264,24 @@ public function crawl_php_file($file)
 
 					// Find event description line
 					$description_line_num = $this->find_description();
-					$description = substr(trim($this->file_lines[$description_line_num]), strlen('* '));
+					$description_lines = array();
+
+					while (true)
+					{
+						$description_line = substr(trim($this->file_lines[$description_line_num]), strlen('*'));
+						$description_line = trim(str_replace("\t", " ", $description_line));
+
+						// Reached end of description if line is a tag
+						if (strlen($description_line) && $description_line[0] == '@')
+						{
+							break;
+						}
+
+						$description_lines[] = $description_line;
+						$description_line_num++;
+					}
+
+					$description = trim(implode(' ', $description_lines));
```

Comment:
implode with space or new line?

Reply:
There's weird font size when using `\n`. I'm voting for `<br/>`.

![linebreaks](https://user-images.githubusercontent.com/6710357/42413695-dcaf765c-8225-11e8-855d-7dbc90192ea1.png)

Reply:
ok